OutOfSchool/OutOfSchool.DataAccess/Repository/ChatMessageRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.ChatWorkshop;
+
+namespace OutOfSchool.Services.Repository;
+
+public class ChatMessageRepository : SensitiveEntityRepository<ChatMessageWorkshop>, IChatMessageRepository
+{
+    private readonly OutOfSchoolDbContext db;
+
+    public ChatMessageRepository(OutOfSchoolDbContext dbContext)
+        : base(dbContext)
+    {
+        db = dbContext;
```

Comment:
(Design issue)
- Mom, could we buy a `dbContext`? (`db = dbContext`)
- No! We have a `dbContext` at home! (Check protected `EntityRepositoryBase.dbContext`)

Please remove excessive field and use inherited one.

Reply:
Fixed. Thank you!